ChangeLog

```diff
@@ -1,3 +1,10 @@
+Release 0.2.11 - 2017-04-23
+
+* Set http_proxy.https option to true by default v2 [#54]
```

Comment:
"v2" should be removed here. That's just added in the title of #54 instead of #52.